cdp_scrapers/legistar_utils.py

```diff
@@ -473,6 +482,30 @@ def get_video_uris(self, legistar_ev: Dict) -> List[Dict]:
         )
         raise NotImplementedError
 
+    def get_time_zone(self) -> str:
```

Comment:
I don't believe this is the correct approach. This and `find_time_zone()` already exist in CouncilDataProject/cdp-scrapers. I think you should merge into yours (fetch upstream). That will bring in the latest, including the time zone related code. May also reveal other conflicts that need resolving, before we can merge this PR into CouncilDataProject/cdp-scrapers.

Reply:
I think this is just a merge conflict oddity, the relevant portion is there. The function required `get_time_zone` has been added to the `KingCountyScraper`